spec/serializers/api/public/v1/serializable_subject_spec.rb

```diff
@@ -0,0 +1,24 @@
+require "rails_helper"
+
+RSpec.describe API::Public::V1::SerializableSubject do
+  let(:non_bursary_subject) { find_or_create :primary_subject, :primary_with_english }
```

Comment:
This is me just nit picking but could we be consistent with methods and their surrounding parenthesis in this spec? 🙏🏽